pkg/webhook/service_labels/types.go

```diff
@@ -48,7 +48,12 @@ type Endpoints struct {
 }
 
 func (ep Endpoints) Namespace() string {
-	return ep.GetNamespace()
+	namespace := ep.GetNamespace()
```

Comment:
maybe let's use named returns to avoid allocation of a new variable, here?

```go
func (ep Endpoints) Namespace() (namespace string) {
  namespace = ep.GetNamespace()
  if len(namespace) == 0 {
    namespace = ep.Subsets[0].Addresses[0].TargetRef.Namespace
  }
  return
}
```

Reply:
Ok, applied and rebased :)